packages/imperative/src/imperative/src/plugins/utilities/npm-interface/install.ts

```diff
@@ -12,6 +12,7 @@
 import { PMFConstants } from "../PMFConstants";
 import * as path from "path";
 import * as fs from "fs";
+import * as jsonfile from "jsonfile";
 import { readFileSync, writeFileSync } from "jsonfile";
```

Comment:
should we remove one of the two?
I think we are using the same two methods 😋 

Or even better, could we try to use `fs` ?

Same for `uninstall`

Reply:
I removed the duplicate import in [b412e4d](https://github.com/zowe/zowe-cli/pull/2015/commits/b412e4dd2fb6d54a8856f90814235c0114470279) 👍 

I will test using `fs.write/read`, not sure if it will format the JSON properly but I'll investigate 😅

Reply:
Per my comment on #2001, going to track this as a separate enhancement since we use `jsonfile` in multiple places outside of this context 😋